docs-website/package.json

```diff
@@ -21,15 +21,18 @@
   "dependencies": {
     "@ant-design/icons": "^4.7.0",
     "@docusaurus/core": "^2.1.0",
+    "@docusaurus/plugin-content-docs": "^2.1.0",
     "@docusaurus/plugin-ideal-image": "^2.1.0",
     "@docusaurus/preset-classic": "^2.1.0",
     "@octokit/plugin-retry": "^3.0.9",
     "@octokit/plugin-throttling": "^3.5.1",
     "@octokit/rest": "^18.6.2",
+    "antd": "^5.0.7",
     "clsx": "^1.1.1",
     "docusaurus-graphql-plugin": "0.5.0",
     "docusaurus-plugin-sass": "^0.2.1",
     "dotenv": "^16.0.1",
+    "fs": "^0.0.1-security",
```

Comment:
@hsheth2 should I be worried about this? I don't think this looks correct 😬

Reply:
yes that doesn't look right - we should be able to remove that since `fs` is built-in